diamm/serializers/website/stats.py

```diff
@@ -0,0 +1,11 @@
+import serpy
+
+from diamm.serializers.serializers import ContextDictSerializer
+
+class StatsSerializer(ContextDictSerializer):
+    source = serpy.IntField(label='sources')
```

Comment:
I was under the impression that the variable name was used to lookup the value in the dict that's passed to the serializer. Wouldn't `sources = serpy.IntField()` result in a KeyError